skycoin/systray
# Fix Data Race

Fixes part of https://github.com/skycoin/skywire/issues/1272.

Changes:
- get new updates from fyne-io
- change URLs

How to test:
- Clone systray at this branch
- Clone skywire, checkout to develop
- Put `replace github.com/skycoin/systray => ../systray` end of `go.mod` of skywire
- Change line 251 of Makefile of skywire to `CGO_ENABLED=1 ${OPTS} go build -race ${BUILD_OPTS} -tags systray -o ./ ./cmd/skywire-visor`
- Build systray apps by `make build-systray`
- Run skywire-visor and check logs for no data race
